models/ai85net_adi_tinierssd.py

```diff
@@ -0,0 +1,461 @@
+###################################################################################################
+#
+# Copyright (C) 2022 Maxim Integrated Products, Inc. All Rights Reserved.
+#
+# Maxim Integrated Products, Inc. Default Copyright Notice:
+# https://www.maximintegrated.com/en/aboutus/legal/copyrights.html
+#
+###################################################################################################
+"""
+Tiny SSD (Single Shot Detector) Variant Model
+"""
+import sys
+from math import sqrt
+
+import torch
+import torch.nn.functional as F
+from torch import nn
+
+import ai8x
+import utils.object_detection_utils as obj_detect_utils
+
+sys.path.append('../')
+
+device = torch.device("cuda" if torch.cuda.is_available() else "cpu")

```

Comment:
This will conflict with the logic in train.py which is more complicated (e.g., supports --cpu)d

Reply:
Done and commit will be amended